src/index.tsx

```diff
@@ -61,6 +62,7 @@ const ManuscriptEditor: React.FC<EditorAppProps> = ({
     }
   }, [authToken])
 
+  setConfig(config)
```

Comment:
can we move it into a useEffect? Especially if we are going to use context to pass it around.

Reply:
Given that I don't expect the config to change and to avoid the overhead of using `useEffect`, I updated the code to use `useMemo`. And regarding using Context, please check my comment above ☝️  and tell me what you think.

Reply:
as things are right now, updating it will not cause any reactive updates and this memo will rerun anytime it's updated anyway, so if we'd like to keep the current approach, we probably wouldn't need it. My suggestion to move it into useEffect (sorry,  I should've specified - the one with the empty deps), was to allow setting config only on component mount. What would be the overhead of using useEffect here?

Reply:
Maybe you’re right, `useMemo` might not be suitable. However, regarding `useEffect`, even with an empty dependency array (which resolves the overhead concern), we will still face a timing issue because it runs after the component is rendered. And as you know, we need to set the configuration before rendering begins.

Reply:
Got it. Thanks for the explanation.